### PR TITLE
Allow managing cert auth secret external to chart

### PR DESCRIPTION
**Pull request by anr-tonic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ generated. If a timestamp is not provided, then the current date and time at
 installation is used even if the key pair has not been updated. Changing this
 date will cause refresh tokens to be invalidated.
 
+Alternatively, a secret with the fields of `public-key` and `creation-date`
+matching the above can be created and the name of the secret provided to
+`web.configuration.masterCert.secretName`.
```
Comment (daniel-tonic) [inline README.md]: Do we want to change masterCert here too?

**Review by daniel-tonic — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (anr-tonic) [reply README.md]: Can. I'd like to keep supporting the `masterCert` as a deprecated field to give values files a chance to catch up for any manual installs.

Comment (daniel-tonic) [reply README.md]: We're not in production yet so I'm not too concerned about it. Otherwise :+1:

Comment (anr-tonic) [reply README.md]: Too many things named tim :shakefist: Yeah you're right

**Pull request merged**